### PR TITLE
BACKLOG-16815 Update project name and signature

### PR DESCRIPTION
**Pull request by AKarmanov**
## JIRA

https://jira.jahia.org/browse/BACKLOG-16815

## Description

Update project name and signature

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@jahia/siteSettings",
```
Comment (tdraier) [inline package.json:2]: it's actually "rolesManager" :-)

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+  "name": "@jahia/roleManager",
```
Comment (gflores-jahia) [inline package.json:2]: should maybe change to `rolesmanager` just to keep consistent with module ID

File: pom.xml [hunk 3]
```diff
@@ -106,55 +106,55 @@
             </resource>
         </resources>
         <plugins>
-            <plugin>
```
Comment (tdraier) [inline pom.xml:109]: why commenting that ?

**Review by gflores-jahia — CHANGES_REQUESTED** [review]

**Review by tdraier — CHANGES_REQUESTED** [review]

**Comment by tdraier** [comment]
@gflores-jahia  same review at same time :-)

File: pom.xml (hunk 3 above)
Comment (AKarmanov) [reply pom.xml]: It sneaked in, I forgot I did that while playing with generate files :( should be fixed now

**Review by gflores-jahia — APPROVED** [review]

**Pull request merged**